cmd/chains.go

```diff
@@ -59,19 +59,19 @@ func filesAdd(ctx *config.Context, dir string) error {
 		byt, err := os.ReadFile(pth)
 		if err != nil {
 			fmt.Printf("failed to read file %s, skipping...\n", pth)
-			continue
+			return err
```

Comment:
Why not convert `fmt.Printf` above into `fmt.Errorf` and return it?
Don't forget to remove "\n" and add the original error using "%v".

Reply:
https://github.com/hyperledger-labs/yui-relayer/pull/105/commits/21850879ea0f2ffc0d019cf16f7bec34305c0b5f
Use `fmt.Errorf`